src/utilities.jl

```diff
@@ -59,6 +59,8 @@ function prettypercent(nominator, denominator)
     else
         str = string(@sprintf("%.2f", value), "%")
     end
-    return lpad(str, 5, " ")
+    # Workaround...
+    str == "100.0%" && (str = "100%")
```

Comment:
A workaround like this is also needed for strings like `"10.00%"`, perhaps check if string ends with `.00%` ? Note that the line below is one char too long:

```
   exact error           20   63.7ms  10.00%  3.19ms   1.35MiB  10.2%  69.2KiB
 ────────────────────────────────────────────────────────────────────────────
```